obs/obs.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"os"
+)
+
+type Project struct {
+	XMLName      xml.Name     `yaml:"project" json:"project" xml:"project"`
+	Name         string       `yaml:"name" json:"name" xml:"name,attr"`
+	Title        string       `yaml:"title" json:"title" xml:"title"`
+	Description  string       `yaml:"description" json:"description" xml:"description"`
+	URL          *string      `yaml:"url" json:"url" xml:"url,omitempty"`
+	Persons      []User       `yaml:"person" json:"persons" xml:"person"`
+	Repositories []Repository `yaml:"repositories" json:"repositories" xml:"repository"`
+	Build        *Build       `yaml:"build" json:"build" xml:"build,omitempty"`
+	Publish      *Publish     `yaml:"publish" json:"publish" xml:"publish,omitempty"`
+	DebugInfo    *DebugInfo   `yaml:"debugInfo" json:"debugInfo" xml:"debuginfo,omitempty"`
+	UseForBuild  *UseForBuild `yaml:"useForBuild" json:"useForBuild" xml:"useforbuild,omitempty"`
+	Parent       *string      `yaml:"parent" json:"parent" xml:"parent"`
+}
+
+type Client struct {
+	Username string
+	Password string
+}
+
+type Path struct {
```

Comment:
Let's call this something like RepositoryPath to make it clear what is it used for